app-emacs/gnuplot-mode/gnuplot-mode-0.7.0.ebuild

```diff
@@ -11,7 +11,7 @@ SRC_URI="https://github.com/bruceravel/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz
 
 LICENSE="GPL-2+"
 SLOT="0"
-KEYWORDS="alpha amd64 arm ~hppa ia64 ppc ppc64 ~s390 sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos"
+KEYWORDS="alpha amd64 arm ~hppa ia64 ppc ppc64 ~s390 ~sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos"
```

Comment:
This is not necessary. sparc is an experimental profile, so the dependency tree need not be complete.
(But again, it's for the sparc team to decide. I would leave the stable keyword in place, though.)

Reply:
I know it is not necessary, but I tried to be nice wrt dep resolution.